providers/jikkou-provider-schema-registry/src/main/java/io/streamthoughts/jikkou/schema/registry/model/Modes.java

```diff
@@ -0,0 +1,17 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ * Copyright (c) The original authors
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.streamthoughts.jikkou.schema.registry.model;
+
+/**
+ * Schema modes
+ */
+public enum Modes {
```

Comment:
All classes that are serialized/deserialized must be annotated with the jikkou `@Reflectable` annotation for graalvm's native image to work correctly.